cmd/internal/cli/instance_stats_linux.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2022, Sylabs Inc. All rights reserved.
```

Comment:
This should almost certainly be your name, or your employer's if it was done under a circumstance in which they claim ownership.